### PR TITLE
Fix check style errors in the code

### PR DESCRIPTION
**Pull request by calebchongsj**

### CODE REVIEW

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -147,18 +150,12 @@ public void setPerson(Person target, Person editedPerson) {
     public void associateOrdersWithPersons() {
         for (CustomerOrder customerOrder : customerOrderList.getOrders()) {
             addressBook.findPersonByPhone(customerOrder.getPerson().getPhone())
-                    .ifPresent(person -> {
-                        customerOrder.setOriginalPerson(person);
-                        person.addOrder(customerOrder);
-                    });
+                    .ifPresent(person -> person.addOrder(customerOrder));
```
Comment (ryenl) [inline src/main/java/seedu/address/model/ModelManager.java:153]: Think setoriginalperson should not be removed as it sets a pointer in the order object to the person object in addressbook during app initialization. Correct me if I am wrong

File: src/main/java/seedu/address/model/ModelManager.java [hunk 2]
```diff
@@ -147,18 +150,12 @@ public void setPerson(Person target, Person editedPerson) {
...
         }
 
         for (SupplyOrder supplyOrder : supplyOrderList.getOrders()) {
             addressBook.findPersonByPhone(supplyOrder.getPerson().getPhone())
-                    .ifPresent(person -> {
-                        supplyOrder.setOriginalPerson(person);
-                        person.addOrder(supplyOrder);
-                    });
+                    .ifPresent(person -> person.addOrder(supplyOrder));
```
Comment (ryenl) [inline src/main/java/seedu/address/model/ModelManager.java:158]: Think setoriginalperson should not be removed as it sets a pointer in the order object to the person object in addressbook during app initialization. Correct me if I am wrong

File: src/main/java/seedu/address/model/ModelManager.java (hunk 1 above)
Comment (calebchongsj) [reply src/main/java/seedu/address/model/ModelManager.java:153]: I am not in charge of this check style code for this class. It should be alright I feel.

File: src/main/java/seedu/address/logic/commands/AddIngredientCommand.java [hunk 3]
```diff
@@ -67,7 +67,8 @@ private boolean isDuplicateIngredient(Map<Integer, Product> catalogue, Ingredien
                 .filter(product -> product instanceof Ingredient) // Filter only Ingredient objects
                 .map(product -> (Ingredient) product) // Cast to Ingredient
                 .anyMatch(existingIngredient ->
-                        existingIngredient.getName().equalsIgnoreCase(newIngredient.getName()));
+                        existingIngredient.getName().equalsIgnoreCase(newIngredient.getName()) &&
+                                Double.compare(existingIngredient.getCost(), newIngredient.getCost()) == 0);
```
Comment (ryenl) [inline src/main/java/seedu/address/logic/commands/AddIngredientCommand.java:71]: Bro you shouldnt add this && cost cus it allows adding two ingredients with same name but diff cost

File: src/main/java/seedu/address/logic/commands/AddIngredientCommand.java [hunk 4]
```diff
@@ -41,7 +41,7 @@ public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
         // Use the existing ingredient catalogue from the model
-        IngredientCatalogue ingredientCatalogue = model.getIngredientCatalogue();
```
Comment (ryenl) [inline src/main/java/seedu/address/logic/commands/AddIngredientCommand.java:44]: this change bugs the ingredient catalogue boss

**Pull request merged**